docs/docs/sourcing-from-forestry.md

```diff
@@ -0,0 +1,185 @@
+
+_Note_: You can see the complete example at [https://github.com/cameron-yee/gatsby-forestry-example](https://github.com/cameron-yee/gatsby-forestry-example).
+
+## Setup
+
+First, clone the Gatsby default starter repository.
+
+```shell
+git clone https://github.com/gatsbyjs/gatsby-starter-default.git gatsby-forestry-example
+```
+
+Next, on GitHub create a new repository. The repository must live on a Git provider for it to be available to Forestry CMS. I'm using GitHub, but you can also use GitLab, Bitbucket, or Azure Devops as well. Set your cloned repository's remote url to point to your newly created repository on GitHub.
+
+```shell
+git remote set-url origin <your-github-repo-link>
+git push -u origin master
+```
+
+Now that your repository is on GitHub, follow the [Forestry.io get started guide](https://forestry.io/docs/quickstart/setup-site/) on Forestry.io to connect the repository to Forestry. You have to allow Forestry access to your GitHub account to make this work. A pop-up window will automatically open when you select your GitHub repo in the Forestry site setup. Once the site is connected, go to the site settings on the Forestry dashboard. In the Admin Path setting, enter "/admin". This will create a static html file for the Forestry CMS login page at the path: /admin.
+
+Once Forestry is connected with your GitHub Gatsby site repository, pull Forestry's changes.
+
+```shell
+git pull
+```
+
+This will add a `.forestry` directory in your project root with the `settings.yml` file. This file allows you to configure your CMS settings including adding content collections. You should also see a new `/admin` directory that contains `index.html`. Create a directory named `static` in your root directory. Move the `/admin` directory into the new `/static` directory. _Important_: The `/admin` directory must be located inside of `/static`. Do not name this directory anything else.
+
+### Settings Configuration
+
+Open `.forestry/settings.yml`. You will see the following default configuration values:
+
+```yaml:title=.forestry/settings.yml
+---
+new_page_extension: md
+auto_deploy: false
+admin_path: /admin
+webhook_url:
+sections:
+upload_dir: uploads
+public_path: "/uploads"
+front_matter_path: ""
+use_front_matter_path: false
+file_template: ":filename:"
+build:
+  preview_output_directory: public
+  install_dependencies_command: npm install
+  preview_docker_image: node:10
+  mount_path: "/srv"
+  working_dir: "/srv"
+  instant_preview_command: npm run forestry:preview
+```
+
+The `new_page_extension` setting can either be `md` or `html`.
+
+Change `upload_dir` and `public path`:
+
+```yaml:title=.forestry/settings.yml
+upload_dir: static/uploads
+public_path: "/static/uploads"
+```
+
+This sets where media is saved. Visit Gatsby's [guide on its static folder](/docs/static-folder/) to learn more. The rest of the settings can remain the same.
+
+Open `package.json` and add the following to scripts:
+
+```json:title=package.json
```

Comment:
If the user is starting with the Gatsby starter then most of this scripts are included. Perhaps it makes sense to just show the forestry:preview script.